sample_network_tests/vane_system_tests/test_vane.py

```diff
@@ -292,3 +293,237 @@ def test_if_setup_fail_is_handled(self):
+            cfg_cmds = ["interface eth16", f"description {description}"]
+
+            """
+            TS: Run cmds using eapi run_cfg_cmds
+            """
+            tops.run_cfg_cmds(cfg_cmds, conn_type="eapi")
+
+            """
+            TS: check if config is present
+            """
+
+            show_cmds = ["show running-config interfaces ethernet 16"]
+
+            tops.actual_output = tops.run_show_cmds(show_cmds, encoding="text")
+
+            tops.expected_output = f"description {description}"
+
+            cfg_cmds = ["interface eth16", f"no description {description}"]
+
+            tops.run_cfg_cmds(cfg_cmds, conn_type="ssh")
+
+        except (AttributeError, LookupError, EapiError) as exception:
+            logging.error(
+                f"On device {tops.dut_name}: Error while running testcase on DUT is: "
+                f"{str(exception)}"
+            )
+            tops.actual_output = str(exception)
+            tops.output_msg += (
+                f"EXCEPTION encountered on device {tops.dut_name}, while "
+                f"investigating if ssh can be used to run cmds. Vane recorded error: {exception}"
+            )
+
+        tops.parse_test_steps(self.test_run_cfg_cmds_eapi_func)
+        tops.test_result = tops.expected_output in tops.actual_output[0]["result"]["output"]
+        tops.generate_report(tops.dut_name, tops.output_msg)
+        assert tops.expected_output in tops.actual_output[0]["result"]["output"]
+
+    def test_file_transfer(self, dut, tests_definitions):
+        """TD: Verifies transfer_file() ssh functionality
+
+        Args:
+          dut (dict): Encapsulates dut details including name, connection
+        """
+
+        tops = tests_tools.TestOps(tests_definitions, TEST_SUITE, dut)
+
+        try:
+            """
+            TS: Transfer file named "sample.txt"
+            """
+            src_file = "sample_network_tests/vane_system_tests/sample.txt"
+            timestr = time.strftime("%Y%m%d-%H%M%S")
+            dest_file = f"sample-{timestr}.txt"
+
+            file_system = "/mnt/flash"
+            operation = "put"
+            tops.expected_output = {
+                "file_exists": True,
+                "file_transferred": True,
+                "file_verified": True,
+            }
+            tops.actual_output = tops.transfer_file(
+                src_file=src_file, dest_file=dest_file, file_system=file_system, operation=operation
+            )
+
+        except (AttributeError, LookupError, EapiError) as exception:
+            logging.error(
+                f"On device {tops.dut_name}: Error while running testcase on DUT is: "
+                f"{str(exception)}"
+            )
+            tops.actual_output = str(exception)
+            tops.output_msg += (
+                f"EXCEPTION encountered on device {tops.dut_name}, while "
+                f"investigating if ssh can be used to run cmds. Vane recorded error: {exception}"
```

Comment:
Need an appropriate output message: 
```suggestion
                f"investigating if file transfer using ssh. Vane recorded error: {exception}"
```

Reply:
Agreed. I did at one point rebase it #382's  branch. But then I change it again. I will be careful.